pom.xml

```diff
@@ -56,6 +56,13 @@
       <version>2.0.4</version>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>matrix-auth</artifactId>
+      <version>1.3</version>
+      <scope>test</scope>
```

Comment:
Yes. I could create Mock ones, but a realistic case slightly improves the test coverage